lnrpc/rpc.proto

```diff
@@ -1669,15 +1669,21 @@ message Invoice {
     */
     uint64 settle_index = 17 [json_name = "settle_index"];
 
+    /// Deprecated, use amt_paid_sat or amt_paid_msat.
+    int64 amt_paid = 18 [json_name = "amt_paid", deprecated = true];
+
     /**
-    The amount that was accepted for this invoice. This will ONLY be set if
-    this invoice has been settled. We provide this field as if the invoice was
-    created with a zero value, then we need to record what amount was
-    ultimately accepted. Additionally, it's possible that the sender paid MORE
-    that was specified in the original invoice. So we'll record that here as
-    well.
+    The amount that was accepted for this invoice, in satoshis. This will ONLY
```

Comment:
Great comment :) can we add the same thing w/ different units on `amt_paid_msat`?

Reply:
Done!